source/style-guide/writing/use-present-tense.txt

```diff
@@ -4,18 +4,20 @@
 Use Present Tense
 =================
 
+.. default-domain:: mongodb
+
 .. include:: /includes/styles/corrections.rst
 
-Users read content to help them perform tasks or to gather information.
-These activities occur in the users' present time, so the present tense
-is appropriate in most content. The present tense implies that users
-should take action now. Sentences that use the present tense are easier
-to read than sentences that use past or future tense.
+Users read content to learn how to perform tasks or to gather
+information. These activities occur in the users' present time. This
+makes the present tense appropriate in most cases. The present tense
+implies that users should take action now. Sentences that use the
+present tense are easier to read than sentences that use past or future
+tense.
 
-The future tense can imply a sense of uncertainty. (We can't truly
-guarantee a future action, regardless of expectation!) Use future tense
-only when you need to emphasize that something occurs later, from the
-users' perspective.
+The future tense may imply a sense of uncertainty. (We can't guarantee
+a future action, regardless of expectation!) Use future tense to
+emphasize that something occurs later from the users' perspective.
```

Comment:
Discussion of past behavior. Server manual often makes use of "previous" to discuss behavior that no longer applies to current builds of MDB:

"Previously, Feature X was limited to Linux deployments"
"Previous to MongoDB 4.4, Kerberos was a fun and enjoyable technology to deploy"

Often uses present tense to discuss, as in this example from our 4.4 change notes (very Ravind comment, I think):

> In previous versions of MongoDB, if you include the { $meta: "textScore" } in both the projection and sort, you must specify the same field name in both places.

Not sure how best to integrate this, but definitely important enough to include. Ping me if you want to talk more about it.

Reply:
Added it right after this block.